src/bench/wallet_loading.cpp

```diff
@@ -71,7 +71,7 @@ static void WalletLoading(benchmark::Bench& bench, bool legacy_wallet)
         options.create_flags = WALLET_FLAG_DESCRIPTORS;
         options.require_format = DatabaseFormat::SQLITE;
     }
-    auto database = CreateMockWalletDatabase(options);
+    auto database = CreateMockableWalletDatabase();
```

Comment:
What about the db `require_format` that is above? Is that used anywhere?

Reply:
Added a commit dropping those.